var/spack/repos/builtin/packages/icu4c/package.py

```diff
@@ -88,3 +104,33 @@ def configure_args(self):
             args.append("--enable-rpath")
 
         return args
+
+
+class MSBuildBuilder(spack.build_systems.msbuild.MSBuildBuilder, BuildEnvironment):
+    def msbuild_args(self):
+        return [
+            "allinone.sln",
+            self.define("OutputPath", self.spec.prefix),
+            self.define("Configuration", "Release"),
+            self.define("SkipUWP", "true"),
+        ]
```

Comment:
I just realized: you added C/C++ flags to MSVC, but I don't see these configure args accounting for the chosen standard. Is the `icu4c` build on Windows customizable in that way?

Reply:
It is evidently not, I'll make the variant non-Windows only.